### PR TITLE
also build SARtools and Envisat tools for Doris

### PR DESCRIPTION
**Pull request by boegel**
follow-up for #1154

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86 please review?

File: easybuild/easyblocks/d/doris.py [hunk 1]
```diff
@@ -73,17 +73,63 @@ def configure_step(self):
...
 
+        super(EB_Doris, self).build_step()
+
+        # build SARtools
+        change_dir(os.path.join(self.cfg['start_dir'], 'SARtools'))
+
+        self.cfg['buildopts'] = common_buildopts
+        self.cfg.update('buildopts', 'CC="%s"' % os.getenv('CXX'))
+        cflags = os.getenv('CFLAGS') + " -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE"
+        self.cfg.update('buildopts', 'CFLAGS="%s"' % cflags)
```
Comment (wpoely86) [inline easybuild/easyblocks/d/doris.py]: `CXXFLAGS`?

File: easybuild/easyblocks/d/doris.py [hunk 2]
```diff
@@ -73,17 +73,63 @@ def configure_step(self):
...
+        self.cfg.update('installopts', 'CFLAGS="%s"' % os.getenv('CFLAGS'))
+        super(EB_Doris, self).install_step()
+
     def sanity_check_step(self):
         """Custom sanity check for Doris."""
         custom_paths = {
-            'files': ['bin/cpx2ps', 'bin/doris', 'bin/plotcpm', 'bin/run'],
+            'files': [os.path.join('bin', x) for x in ['cpx2ps', 'doris', 'plotcpm', 'run']] +
+                     [os.path.join('bin', x) for x in ['bkconvert', 'cpxfiddle', 'flapjack', 'floatmult', 'wrap']] +
+                     [os.path.join('bin', x) for x in ['envisat_dump_header', 'envisat_dump_data']],
```
Comment (wpoely86) [inline easybuild/easyblocks/d/doris.py]: why not put them all in a list variable outside this dict?

File: easybuild/easyblocks/d/doris.py (hunk 1 above)
Comment (boegel) [inline easybuild/easyblocks/d/doris.py]: well, I need to define `CFLAGS`, but I should use the `$CXXFLAGS` value as defined by EasyBuild here, yes

File: easybuild/easyblocks/d/doris.py (hunk 2 above)
Comment (boegel) [inline easybuild/easyblocks/d/doris.py]: I grouped them by origin (Doris vs SARtools vs ENVISAT_TOOLS), I'll include a comment to make that clear

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**